emukit/model_wrappers/gpy_quadrature_wrappers.py

```diff
@@ -162,26 +164,57 @@ def dKdiag_dx(self, x: np.ndarray) -> np.ndarray:
         return np.zeros((input_dim, num_points))
 
 
-def convert_gpy_model_to_emukit_model(gpy_model, integral_bounds: List[Tuple[float, float]], integral_name: str = '') \
+def convert_gpy_model_to_emukit_model(gpy_model, integral_bounds: Union[None, List[Tuple[float, float]]],
```

Comment:
Since the type of the model is known, feel free to use it as a type hint here.

Also, semantically "create_" would be more correct than "convert_to_" - wrapping is not the same as converting

Reply:
I added the type hint and renamed the function to 'create_emukit_model_from_gpy_model'